r/aws-securityhub: Add aws_securityhub_account resource (#6839)
terraform-providers/terraform-provider-aws: website/docs/r/securityhub_account.markdown

---
layout: "aws"
page_title: "AWS: aws_securityhub_account"
sidebar_current: "docs-aws-resource-securityhub-account"
description: |-
  Enables Security Hub.
---

# aws_securityhub_account

> -> **Note:** Destroying this resource will disable Security Hub.

Reviewer: Nit: Should probably mention just for this AWS account. 😄 

```suggestion
-> **Note:** Destroying this resource will disable Security Hub for this AWS account.
```

> Enables Security Hub.

Reviewer: Nit: Should probably mention just for this AWS account. 😄 

```suggestion
Enables Security Hub for this AWS account.
```

## Example Usage

```hcl

> resource "aws_securityhub_member" "example" {}

Reviewer: 🍝 Copy-pasta

```suggestion
resource "aws_securityhub_account" "example" {}
```

```

## Argument Reference

The resource does not support any arguments.

## Attributes Reference

The following attributes are exported in addition to the arguments listed above:

> * `id` - Returns `securityhub-account`.

Reviewer: ```suggestion
* `id` - AWS Account ID
```

## Import

> Security Hub account enablemenet can be imported using the word `securityhub-account`, e.g.

Reviewer: ```suggestion
An existing Security Hub enabled account can be imported using the AWS account ID, e.g.
```

```

> $ terraform import aws_securityhub_account.example securityhub-account

Reviewer: ```suggestion
$ terraform import aws_securityhub_account.example 123456789012
```
